contracts/security-token/BasicKYC.sol

```diff
@@ -1,13 +1,14 @@
 pragma solidity ^0.4.18;
 
+import "./KYCInterface.sol";
 import "zeppelin/contracts/ownership/Ownable.sol";
 
-contract MockKYC is Ownable {
+contract BasicKYC is Ownable, KYCInterface {
```

Comment:
I think BasicKYC needs some more comments to explain the `nonce` thing... if there is anything magical or not.

Also `whitelistMe` is not obvious to external reader. We can say that the signature comes from the server side.